x-pack/docs/en/security/securing-communications/security-basic-setup.asciidoc

```diff
@@ -0,0 +1,277 @@
+[[security-basic-setup]]
+=== Set up basic security for the Elastic Stack
+++++
+<titleabbrev>Set up basic security</titleabbrev>
+++++
+
+After adding password protection in the <<security-minimal-setup,minimal security configuration>>, you'll need to configure Transport Layer Security
+(TLS). The transport layer handles all internal communication between nodes in
+your cluster.
```

Comment:
This is a fantastic explanation @tvernum! I'm incorporating this information into the introduction for the basic security scenario.

Reply:
Distinguishing the two usages of the word "transport" is a pain indeed. The best way I have found to describe the transport interface/protocol/layer/port/module/... is to always contrast it to the HTTP interface as suggested here. I also think it's a good idea to call out the unfortunate overloading of the word "transport".

We don't call it the "transport layer" in https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-network.html, so at least we could avoid "layer" here too where we're not talking about TLS. (I know we say "layer" a couple of times near the bottom of that page but that's a fixable bug). We use the phrase "transport interface" in those docs, but that's because we're mostly talking about the endpoint on one specific node rather than anything more general. Perhaps that's a useful distinction here too, but if not then I don't have a great alternative noun. Definitely not "layer" anyway, and it would be good to avoid lots of different words for the same thing.

I think it would help also to contrast more clearly the security effects of applying TLS to the the transport vs HTTP interfaces. Not sure if this is the best place for that detail but a link to that detail from here would be good.

Reply:
@DaveCTurner, thanks for weighing in. Does it make sense to incorporate the section I added for [Understanding transport contexts](https://elasticsearch_68946.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/security-basic-setup.html) to the Networking page you linked to, and then link users to that page from these TLS docs?

Reply:
> Does it make sense to incorporate the section I added for Understanding transport contexts to the Networking page you linked to

No, there's nothing else on that page about TLS so I don't think it makes sense to put that text there.

Reply:
Thanks @DaveCTurner -- I think we can leave this discussion of transport where it is for now, and revisit whether it belongs somewhere else in a separate PR.